src/shared/bindings.ts

```diff
@@ -57,15 +57,12 @@ export class BindingUtils {
     return bindingUserSettingsMetaData;
   }
 
-  public static getHttpOutBinding(bindingUserSettings) {
+  public static getHttpOutBinding() {
     const binding = {};
 
     binding[constants.type] = "http";
     binding[constants.direction] = constants.outDirection;
-    binding[constants.name] = "$return";
-    if (bindingUserSettings[constants.webHookType]) {
-      binding[constants.name] = "res";
-    }
+    binding[constants.name] = "res";
```

Comment:
Do we always want it to be res, or just default to it and still have the option of choosing the $return?

Reply:
I need the option of choosing $return instead of res.